sdks/agent-sdk/README.md

```diff
@@ -89,11 +89,15 @@ Subscribe only to what you need using Node’s `EventEmitter` interface. Events
 **Message Events**
 
 - `attachment` – an incoming [remote attachment message](https://docs.xmtp.org/chat-apps/content-types/attachments)
+- `markdown` – an incoming [markdown-formatted](https://docs.xmtp.org/agents/content-types/markdown) text message
 - `message` – all messages that are not having a [custom content type](https://docs.xmtp.org/agents/content-types/content-types#custom-content-types)
 - `group-update` – an incoming [group update](https://docs.xmtp.org/agents/content-types/group-updates#listen-for-group-updates) (like name change, member update, etc.)
 - `reaction` – an incoming [reaction message](https://docs.xmtp.org/agents/content-types/reactions)
+- `read-receipt` – an incoming [read receipt](https://docs.xmtp.org/chat-apps/content-types/read-receipts) notification
 - `reply` – an incoming [reply message](https://docs.xmtp.org/agents/content-types/replies)
 - `text` – an incoming [text message](https://docs.xmtp.org/agents/content-types/content-types#text-content-type)
+- `transaction-reference` – an incoming onchain [transaction reference](https://docs.xmtp.org/agents/content-types/transaction-refs#receive-a-transaction-reference)
+- `transaction-request` – an incoming wallet [transaction request](https://docs.xmtp.org/agents/content-types/transactions#create-a-transaction-request) (batch calls)
```

Comment:
while i think this is more semantic name, i think we should stick to the names of the content types for consistency. in this case: `wallet-send-calls`.

Reply:
Ok, I reverted the renaming... https://github.com/xmtp/xmtp-js/pull/1471/commits/e01fce92b04cc997a2c90751a0bd262306ef8502

We can add `transaction-request` additionally if we think it is a better fit.